templates/checkout/_partials/steps/shipping.tpl

```diff
@@ -46,7 +46,7 @@
                   </div>
                 </label>
 
-                <div class="delivery-option__extra {if $delivery_option == $carrier_id} delivery-option__extra--active {/if} js-carrier-extra" {if $delivery_option == $carrier_id} data-active{/if}>
+                <div class="delivery-option__extra js-carrier-extra" {if $delivery_option == $carrier_id}data-active{/if}>
```

Comment:
@tblivet any reason for the removal of this class? I think that if it existed before, we could keep it for backward compatibility purpose